src/utils.js

```diff
@@ -11,3 +11,19 @@ export function isEnterpriseLearner(user) {
 
   return false;
 }
+
+// eslint-disable-next-line import/prefer-default-export
+export function getPreferredEnterpriseUUID(user) {
+  if (user !== null && user.roles) {
```

Comment:
so i just learned about a JS operator through doing a PR review that helps with these sorts of conditionals. try `if (user?.roles)`! 😄

Reply:
Umm what?! That's hype.

Reply:
And also unavailable until we swap to webpack 5 ☹️